media-libs/dav1d/dav1d-0.5.2.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="A new AV1 cross-platform decoder"
+DESCRIPTION="dav1d is a new AV1 cross-platform decoder, open-source, \
+and focused on speed and correctness. It supports all features from AV1, \
+including all subsampling and bit-depth parameters."
+HOMEPAGE="https://code.videolan.org/videolan/dav1d"
+COPYRIGHT="2018-2019, VideoLAN and dav1d authors"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://code.videolan.org/videolan/dav1d/-/archive/$portVersion/dav1d-$portVersion.tar.bz2"
+CHECKSUM_SHA256="1780199d71e62a82ca1bfca2d71da5c599c7e1edc68c72f6fc3c0c25ce1880b7"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64 ?arm ?ppc"
+SECONDARY_ARCHITECTURES="x86"
+
+commandBinDir=$binDir
+commandSuffix=$secondaryArchSuffix
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+libVersion="3.1.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	dav1d$secondaryArchSuffix = $portVersion
+	cmd:dav1d
+	lib:libdav1d$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	dav1d${secondaryArchSuffix}_devel = $portVersion
+	devel:libdav1d$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	dav1d$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:meson >= 0.47
+	cmd:nasm >= 2.13.02
+	cmd:ninja
+	"
+
+defineDebugInfoPackage dav1d \
```

Comment:
Missed this one:
```diff
- defineDebugInfoPackage dav1d \
+ defineDebugInfoPackage dav1d$secondaryArchSuffix \
```

Reply:
Strike that, there isn't any debug information in the command (nor in the library), you can remove the defineDebugInfoPackage all together :)

Reply:
Thanks, fixed that.

Reply:
If you want to know if a binary/library adds debug information (as @alaviss mentioned) you can also check the generated files with "readelf -S <binary/library> | grep debug" @trungnt2910